DeAceroDLabs/marketplace-frontend
# fix: [DMP-752] Signup error button goes now to /signup

## What is done

[[DMP-696] click on little carousel images](https://deacero.atlassian.net/browse/DMP-667)

## How is done

- Added errorType to signup, now signup is able to see if the error is caused because the user has been already registered before or other type of error.
- It is checked if the email already exists and also rfc.
- Get all users with getUsers endpoint.
- Signup message now has 2 different type of errors,.

## How to test

1. Run the project via `npm start`.
2. Try to register with email: dbasilio@deacero.com you should see this:
3. Try register with other email and rfc that is not registered yet, you should see this:


.
